packages/manager/apps/dedicated/client/app/billing/autoRenew/autorenew.html

```diff
@@ -231,17 +231,8 @@
                 data-type-options="$ctrl.filtersOptions.status"
                 data-filterable
             >
-                <span
-                    class="oui-badge oui-badge_error"
-                    data-ng-if="$row.hasDebt()"
-                    data-translate="billing_autorenew_service_status_pending_debt"
```

Comment:
**thought:** Translation could be removed then.

Reply:
Translations might be still used here https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/billing/autoRenew/autorenew.service.js#L118 but I'll check for the hub translations